source/crud/crud.rst

```diff
@@ -199,17 +199,25 @@ Read
      * If true, allows the write to opt-out of document level validation. This only applies
      * when the $out stage is specified.
      * 
-     * On servers >= 3.2, the default is no value: no 
-     * "bypassDocumentValidation" option is sent with the "aggregate" command.
-     *
-     * On servers < 3.2, this option is ignored.
+     * This option is sent only if the caller explicitly provides a value. The default is to not send a value.
+     * For servers < 3.2, this option is ignored and not sent as documentation validation is not available.
      */
     bypassDocumentValidation: Optional<Boolean>;
 
     /**
+     * Optionally specifies a collation to use in MongoDB 3.4 and higher.
+     *
+     * This option is sent only if the caller explicitly provides a value. The default is to not send a value.
+     * For servers < 3.4, the driver MUST raise an error if the caller explicitly provides a value.
```

Comment:
Yes. Not using a collation when the user specified one could result in deleting the wrong documents, for example.